LICENSE

```diff
@@ -1,191 +1,14 @@
-Apache License
-Version 2.0, January 2004
-http://www.apache.org/licenses/
 
-TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+Copyright (c) 2021-2023, Inversoft Inc.
```

Comment:
I think this file was added in `2014` if we want to be specific, and I think the repo goes back quite a bit further.

Reply:
Updated to 2012